src/Pages/DiscoverRoversPage/DiscoverRoversPage.scss

```diff
@@ -1,15 +1,17 @@
 ﻿.discoverRovers {
-  margin-top: 150px;
   display: flex;
   flex-direction: column;
   align-items: center;
   text-align: center;
+  font-size : large;
+
   h1 {
-    font-size: 2.5rem;
+    font-size: larger;
   }
```

Comment:
Why do we not want to just use the global defaults here?

Reply:
the styling for h1 is far too small